cvat/settings/base.py

```diff
@@ -386,11 +394,16 @@ def add_ssh_keys():
         },
         'logstash': {
             'level': 'INFO',
-            'class': 'logstash.TCPLogstashHandler',
+            'class': 'logstash_async.handler.AsynchronousLogstashHandler',
+            'formatter': 'logstash',
+            'transport': 'logstash_async.transport.HttpTransport',
+            'ssl_enable': False,
+            'ssl_verify': False,
             'host': os.getenv('DJANGO_LOG_SERVER_HOST', 'localhost'),
-            'port': os.getenv('DJANGO_LOG_SERVER_PORT', 5000),
+            'port': os.getenv('DJANGO_LOG_SERVER_PORT', 8080),
```

Comment:
What is the reason to change the port from 5000 to 8080?

Reply:
For http protocol, port 8080 is standard one, and logstash now works via http because of persistent queue

Reply:
@azhavoro , we use 8080 for cvat_proxy. Can it be a problem?

Reply:
This is not open port, I don't see any problem here. We use the same port number for cvat_server https://github.com/openvinotoolkit/cvat/blob/develop/Dockerfile#L157